engine/language-server/src/main/scala/org/enso/languageserver/requesthandler/file/CreateFileHandler.scala

```diff
@@ -35,13 +35,18 @@ class CreateFileHandler(requestTimeout: FiniteDuration, fileManager: ActorRef)
     cancellable: Cancellable
   ): Receive = {
     case Status.Failure(ex) =>
-      log.error(s"Failure during $CreateFile operation:", ex)
+      log.error(
+        ex,
+        "Failure during [{}] operation: {}",
+        CreateFile,
+        ex.getMessage
```

Comment:
Could the exception message contain sensitive data? The same question for the other exception messages we log.

Reply:
Right, IO errors can potentially leak paths inside the user project. I'll mask them as well